front/package.json

```diff
@@ -37,7 +37,8 @@
     "majestic": "majestic --port 4231",
     "find-unused-messages": "node internals/scripts/find-unused-messages.js",
     "prettier": "prettier",
-    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx app --color --max-warnings=0"
+    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx app --color --max-warnings=0",
+    "detect-deadcode": "ts-prune -p app/tsconfig.json -i '(node_modules|utils/testUtils/rtl.tsx|services/permissions|i18n|/*/*/__mocks__)' | (! grep -v 'used in module')"
```

Comment:
Good job! It's so great some languages let you do it!

What is `used in module`? I can find it neither in this PR nor in the repo.

Reply:
Yes, good question! This is found in the output of the tool, actually. `ts-prune` detects when a type is exported but never used elsewhere in the app so it tells you it's only used within the module. I am filtering this out because I don't think it's a huge problem/ worth having as a strict rule.

Reply:
Makes sense. Thanks for the explanation.